### DISCUSSION TITLE
SideBar item's active state

### DISCUSSION THREAD

**Discussion by akil-rails** [vuejs/vitepress#1287] [Q&A] [answered]
Is there a way to open a collapsed side bar, and change the state to active; in cases when such a page is opened via an incoming link from another page on the site? 

Something like what's available for the [Nav Bar](https://vitepress.vuejs.org/guide/theme-nav#customize-link-s-active-state)?

**Comment by brc-dd** [comment, marked as answer]
This should be happening by default in the latest version. This was added in #1094, or are you asking something else?

**Reply by akil-rails** [reply]
Yes; took the latest version & it is working .
